### PR TITLE
3 column homepage

### PR DESCRIPTION
**Pull request by lily-pytel**
Desktop:
image

Looks a little strange on a smaller screen, but not too bad:
image

### CODE REVIEW

**Comment by lily-pytel** [comment]
Addresses #260

File: next/components/author-link.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { ThemeContext } from './theme-context';
+
+export default function AuthorLink({ author }) {
+  const theme = useContext(ThemeContext);
+  const link = author
+    ? `/decks?updatedTime=100000&authorName=${author}&sortBy=hot`
```
Comment (alexmiddeleer) [inline next/components/author-link.js:8]: Does updatedTime need to be included? Eventually this might cause a bug report 😄

Comment (lily-pytel) [reply next/components/author-link.js:8]: Otherwise it shows only the last 150 days 🤷

**Comment by lily-pytel** [comment]
LMK whether we want to do something else with the updated param. Perhaps by the time it's a problem, we'll have profile pages.

**Pull request merged**